code/modules/telesci/gps.dm

```diff
@@ -104,4 +104,9 @@ var/list/GPS_list = list()
 	flags = ABSTRACT
 	gpstag = "Eerie Signal"
 	desc = "Report to a coder immediately."
-	invisibility = INVISIBILITY_MAXIMUM
\ No newline at end of file
+	invisibility = INVISIBILITY_MAXIMUM
+
+/obj/item/device/gps/mining/internal
+	icon_state = "gps-m"
+	gpstag = "MINER"

```

Comment:
This is better because I can use it to powergame on station with unremovable suit sensors.